client/my-sites/hosting/staging-site-card/use-has-valid-quota.ts

```diff
@@ -3,7 +3,9 @@ import wp from 'calypso/lib/wp';
 
 export const USE_VALID_QUOTA_QUERY_KEY = 'valid-quota';
 
-export const useHasValidQuotaQuery = ( siteId: number, options: UseQueryOptions ) => {
+type HasValidQuotaOptions = Pick< UseQueryOptions, 'enabled' >;
```

Comment:
This is a drive-by change. We were never exposing the complete `UseQueryOptions` so we shouldn't type it like that.